### PR TITLE
PreThumbs Stub complete.

### PR DESCRIPTION
**Pull request by JTMcDuff**
Also updated the eslint to only warn on the return assignments.

//  TODO:  Write functions to emit and receive data.
  // Refactor to use Store
  // Find graphics.

### CODE REVIEW

File: client/src/components/PresThumbs.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <div>Thumbs up! {this.thumbsUpTotal}</div><div>Thumbs to the side! {this.thumbsSideTotal}</div><div>Thumbs Down! {this.thumbsDownTotal}</div>
+        </div>
+        <h1>  Images go Here!</h1>
+      </div>
+    );
+  };
+
+};
+
+export default connect(state => state)(PresThumbs);
```
Comment (rtopol2) [inline client/src/components/PresThumbs.jsx]: should use mapStateToProps:

```
const mapStateToProps = (state) => {
  return {
    thumbs: state.thumbs
  };
};

export default connect(mapStateToProps)(PresThumbs);
```

**Comment by sdbedi** [comment]
Looks good.

File: client/src/components/PresThumbs.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+//  TODO:  Write functions to emit and receive data.
+  // Refactor to use Store
+  // Find graphics.
+class PresThumbs extends Component {
+  constructor () {
+    super();
+      // Setting up thumbs variables here so they can be passed and rendered later.
+      // Remove from state when necessary.
+    this.thumbsUpTotal = 0;
```
Comment (rtopol2) [inline client/src/components/PresThumbs.jsx]: these need to be:

```
this.props.thumbs.up
this.props.thumbs.down
this.props.thumbs.side
```

Comment (rtopol2) [inline client/src/components/PresThumbs.jsx]: Actually, nvm, you don't need these here. They'll be grabbed from the store.

File: client/src/components/PresThumbs.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  render () {
+    return (
+      <div>
+        <h1> Topic: </h1>
+        <form>
+          <input type='text' name="topic" /><br/>
+       </form>
+       <button onClick={ console.log('Write a function for me!') }>Set Topic</button>
+        <div>
+        <div>Thumbs up! {this.thumbsUpTotal}</div><div>Thumbs to the side! {this.thumbsSideTotal}</div><div>Thumbs Down! {this.thumbsDownTotal}</div>
```
Comment (rtopol2) [inline client/src/components/PresThumbs.jsx]: for each of these, you should just do:

`{this.props.thumbs.up || 0}`
`{this.props.thumbs.down || 0}`
`{this.props.thumbs.side || 0}`

**Comment by JTMcDuff** [comment]
PR updated.

**Pull request merged**